src/components/molecules/Carousel/carousel.scss

```diff
@@ -0,0 +1,59 @@
+@use '../../../assets/scss/functions/_base.functions.scss' as *;
+@use '../../../assets/scss/variables/variables.module';
+
+.scene {
+  margin: 16px 0;
+  position: relative;
+  perspective: 1000px;
+}
+
+.carousel {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  transform: translateZ(-288px);
+  transform-style: preserve-3d;
+  transition: transform 1s;
+}
+
+.carousel__element {
+  position: absolute;
+  left: 0px;
+  top: 0px;
+  // Si saco este borde, tengo un bug con el border de la badge -- raaaro
```

Comment:
  👀👀

Reply:
Yep, It was just a reminder for me, I need to remove it... Good catch